app/controllers/facility_journals_controller.rb

```diff
@@ -145,6 +145,10 @@ def unreconcile
       raise CanCan::AccessDenied, I18n.t("controllers.facility_journals.unreconcile.feature_disabled")
     end
 
+    unless current_user.administrator?
```

Comment:
there's an `:unreconcile, OrderDetail` rule in abilities which seems to be only enabled to administrators, can we use that?